app/Battle/preparation_battle.py

```diff
@@ -0,0 +1,23 @@
+from app.Knights.knight_class import Knight
+
```

Comment:
Rename your module to `preparation_functions.py` or another name to be more descriptive of its content.

Reply:
👍